docs/zkapps/faq.mdx

```diff
@@ -1,5 +1,12 @@
 ---
 title: FAQ
```

Comment:
```suggestion
title: SnarkyJS FAQ
```

Reply:
This will change the sidebar text for this page from `FAQ` -> `SnarkyJS FAQ`. Is that change intended?

Reply:
oh hmmm can we leave the title as SnarkyJS FAQ and have the sidebar FAQ?

Reply:
Oh, I found out how and propose in the suggestion to add

`sidebar_label: FAQ`